core/src/main/java/de/danoeh/antennapod/core/util/ChapterUtils.java

```diff
@@ -144,8 +137,6 @@ public static void readOggChaptersFromPlayableStreamUrl(Playable media) {
                 if (input != null) {
                     readOggChaptersFromInputStream(media, input);
                 }
```

Comment:
MalformedURLException is a subclass of IOException and the catch clauses are doing exactly the same